addons/event_crm/models/event_registration.py

```diff
@@ -148,7 +148,7 @@ def _update_leads(self, new_vals, lead_tracked_vals):
                 if not lead.partner_id:
                     lead_values['description'] = lead.registration_ids._get_lead_description(_("Participants"), line_counter=True)
                 elif new_vals['partner_id'] != lead.partner_id.id:
-                    lead_values['description'] = lead.description + "<br/>" + lead.registration_ids._get_lead_description(_("Updated registrations"), line_counter=True, line_suffix=_("(updated)"))
+                    lead_values['description'] = lead.description or '' + "<br/>" + lead.registration_ids._get_lead_description(_("Updated registrations"), line_counter=True, line_suffix=_("(updated)"))
```

Comment:
Careful with that:
```python
>>> False or '' + "<br/>"
'<br/>'
>>> 'Hello' or '' + "<br/>"
'Hello'
```
```suggestion
                    lead_values['description'] = (lead.description or '') + "<br/>" + lead.registration_ids._get_lead_description(_("Updated registrations"), line_counter=True, line_suffix=_("(updated)"))
```

Reply:
Wasn't sure and then forgot to check ^^'
Thanks!